app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
```

Comment:
- [ ] This should be an Event model, not Post. Make sure to update it and your migrations, controllerss, etc.